packages/mco/utils/acm.ts

```diff
@@ -65,3 +74,23 @@ export const getClustersFromDecisions = (
 export const getRemoteNamespaceFromAppSet = (
   application: ArgoApplicationSetKind
 ): string => application?.spec?.template?.spec?.destination?.namespace;
+
+export const getLastAppDeploymentClusterName = (
+  drPlacementControl: DRPlacementControlKind
+) =>
+  getAnnotations(drPlacementControl)?.[
+    LAST_APP_DEPLOYMENT_CLUSTER_ANNOTATION
+  ] || '';
+
+export const findDeploymentClusters = (
+  placementDecision: ACMPlacementDecisionKind,
+  drPlacementControl: DRPlacementControlKind
+): string[] => {
+  if (placementDecision?.status.decisions.length > 0) {
```

Comment:
do we need to do this or its guaranteed that if "status" is present then "decisions" will be present as well ??
```suggestion
  if ((placementDecision ?? {}).status?.decisions?.length > 0) {
```

Reply:
ack